curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cb2ee9d60f205d362b1de.md

```diff
@@ -0,0 +1,63 @@
+---
+id: 656cb2ee9d60f205d362b1de
+title: Task 89
+challengeType: 19
+dashedName: task-89
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Yeah. Here in the company, they are very attentive to these details. You are going to like it here if you are into cutting-edge gadgets.
+Tom: This is so cool. A Standing Desk, an ergonomic chair and an ergonomic mouse. Man, everything is perfect. I'm in love with this place!
+-->
+
+# --description--
+
+In workplaces, we often have items that help us do our job better. Both Tom and Sophie talk about items at their desks which make their work more comfortable.
+
+# --instructions--
+
+Listen to this part of Sophie and Tom's conversation and answer the question below.
+
+# --question--
+
+## --text--
+
+Which of these is NOT a workplace item?
+
+## --answers--
+
+`Standing desk`
+
+### --feedback--
+
+`Cutting-edge gadgets` is a general term used by Sophie to describe the latest technology or tools. The other options are specific items Tom lists when describing his workspace. Look for the exact items mentioned by Tom.
+
+---
+
+`Ergonomic chair`
+
+### --feedback--
+
+`Cutting-edge gadgets` is a general term used by Sophie to describe the latest technology or tools. The other options are specific items Tom lists when describing his workspace. Look for the exact items mentioned by Tom.
+
+---
+
+`Ergonomic mouse`
+
+### --feedback--
+
+Feedback3.
+
+---
+
+`Cutting-edge gadgets`
+
+### --feedback--
+
+`Cutting-edge gadgets` is a general term used by Sophie to describe the latest technology or tools. The other options are specific items Tom lists when describing his workspace. Look for the exact items mentioned by Tom.
+
+## --video-solution--
+
+4
```

Comment:
```suggestion
Which of these is NOT a workplace item mentioned by Sophie and Tom?

## --answers--

`Standing desk`

### --feedback--

Tom mentions this workplace Item.

---

`Ergonomic chair`

### --feedback--

Tom mentions this workplace Item.

---

`Ergonomic mouse`

### --feedback--

Tom mentions this workplace Item.

---

`Cutting-edge gadgets`

## --video-solution--

4
```

I noticed the question in this challenge wasn't specific enough. I've also noticed the feedbacks were wrong. I've fixed that.